tssc/step_implementers/create_container_image/__init__.py

```diff
@@ -1,5 +1,26 @@
-"""
-tssc.StepImplementers for the 'create_container_image' TSSC step.
+"""tssc.StepImplementers for the 'create-container-image' TSSC step.
+
+Step Configuration
+------------------
+All tssc.StepImplementers for this step should
+accept minimally the following configuration options.
+
+| Parameter       | Description
+|-----------------|------------
+| `imagespecfile` | Image specification file name
+| `context`       | Parent path to the image specification file
+| `tlsverify`     | Verify TLS certs when pulling images?
+
+Results
+-------
+All tssc.StepImplementers for this step should
+minimally produce the following step results.
+
+| Result Key       | Description
+|------------------|------------
+| `image-tag`      | The image ID to tag the built image with when pushing it to a local file
+| `image-tar-file` | Path to the built container image as a tar file
+
```

Comment:
@itewk Did you want to place an example here like you did for the previous steps?